development/metamaskbot-build-announce.js

```diff
@@ -2,7 +2,7 @@
 const { promises: fs } = require('fs')
 const path = require('path')
 const fetch = require('node-fetch')
-const VERSION = require('../dist/chrome/manifest.json').version // eslint-disable-line import/no-unresolved
+const VERSION = require('../dist/chrome/manifest.json').version // eslint-disable-line import/no-unresolved,node/no-missing-require
```

Comment:
Are these two rules doing the same thing? :thinking:

Reply:
Yes, it seems they do. `node/no-missing-require` and `node/no-missing-import` perform the same checks as `import/no-unresolved`. I'll leave this for now, and create a `@metamask/eslint-config` issue to turn one of these off.

Reply:
https://github.com/MetaMask/eslint-config/issues/74

Reply:
We can drop this if we update to 4.1.0